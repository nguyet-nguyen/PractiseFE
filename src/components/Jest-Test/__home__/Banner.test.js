import BannerHomepage from "pages/LandingPages/Home/sections/Banner";
import React from "react";
import ReactDOM from "react-dom";
import {act} from 'react-dom/test-utils';


describe("Unit Test banner home Component render", () => {
    let container;

    beforeEach(() => {
        container = document.createElement('div')
        document.body.appendChild(container)
    })

    afterEach(() => {
        document.body.removeChild(container)
        container = null;
    })

    it("SnapShot case 1",  () => {
 
        act(() => { 
            ReactDOM.render(<BannerHomepage/>, container) 
        }) 
 
        const button = container.querySelector("div")
        expect(button).toMatchSnapshot()
    }) 

})