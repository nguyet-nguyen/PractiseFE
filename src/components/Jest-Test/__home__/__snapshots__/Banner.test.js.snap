// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit Test banner home Component render SnapShot case 1 1`] = `
<div
  class="container mx-auto flex justify-center"
>
  <div
    class="relative overflow-hidden bg-no-repeat bg-cover w-full"
  >
    <img
      alt="Louvre"
      class="max-w-full h-auto hover:scale-110 transition duration-300 ease-in-out"
      src="banner.JPG"
    />
  </div>
</div>
`;
